### DISCUSSION TITLE
中英文混合检测结果较差

### DISCUSSION THREAD

**Discussion by MonolithFoundation** [PaddlePaddle/PaddleOCR#13206] [Q&A]
"value": "心地带，是该地矿业遗址的缩影。墙体一端高于另一端，象征着该地区经挤压\n形成的独特地貌。墙体是由一个长30m的胶合板模架和一块橡胶长薄板构筑形\n成。首先将薄板进行褶皱和折曲，制成大型的胶状模具，以便将混凝土浇灌于\n其中。然后将模架切割成6个部分，并运往混凝土灌注入模的莱德混凝土厂。\n混凝土凝固后，将其从模具中取出，并运往不到1.6km处的核心区进行安装。\nHeartlandssite.The site's formermining heritage isreflected in thewall.Higher at\noneend than theother.thewallreflectsthecrushingaspectsofgeology.Processof\nwascrumpledandfoldedtocreateagiantjellymouldinwhichtheconcretewas\nset.Formworkwasthencutintosixsectionsand transported toLaddsConcrete\nwhere the concretewaspoured into themoulds.Onceconcretewassetandhad\nbeenremoved from the moulds.it was transported less than1.6 km up the road to the\nHeartlandssiteforinstallation"

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/168061642/b780bbb8-9300-41d7-8731-fd026e060981)

主要问题：

1. 英文有事哦呼有空格，有时候没有；
2. 英文会漏掉

用的是 ch v4模型。

**Comment by GreatV** [comment]
混合场景你可能需要自己再训练一下。
